devstack/plugin.sh

```diff
@@ -39,7 +39,6 @@ function nfp_configure_neutron {
     iniset $NEUTRON_CONF group_policy_implicit_policy default_ip_pool "11.0.0.0/8"
     iniset $NEUTRON_CONF group_policy_implicit_policy default_proxy_ip_pool "192.169.0.0/16"
     iniset $NEUTRON_CONF group_policy_implicit_policy default_external_segment_name "default"
-    iniset $NEUTRON_CONF device_lifecycle_drivers drivers "haproxy, vyos"
```

Comment:
Why is this removed? Shouldn't we add nfp vendor to this list rather than removing it?